pkgs/top-level/python-packages.nix

```diff
@@ -11890,6 +11890,20 @@ in modules // {
     };
   });
 
+  libgpuarray-cuda = callPackage ../development/python-modules/libgpuarray/cuda/default.nix rec {
+    numpy = self.numpy_1_10;
+    scipy = self.buildScipyPackage rec {
+      version = "0.17.1";
```

Comment:
please do not specify specific versions in `python-packages.nix` but use the default version, so `numpy` and `scipy`, since otherwise we get trouble with multiple versions in a closure.